doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -44,6 +44,26 @@ Examples
 ========
 
 Below are examples of a variety two dimensional beam bending problems.
+ 
+.. note::
+
+    There is no specific sign convention that must be followed.
+    Given a right handed coordinate system with XYZ coordinates,
+    the beam's length is assumed to be along the positive X axis.
+    The loading forces are directed either in the positive or
+    negative Y direction and the loading moments are about
+    the positive or negative Z direction.
+    The only thing that should be considered while choosing
+    sign onvention is that forces acting along positive
+    Y direction(upward) and moments acting along
+    negative z direction(clockwise) must have the same signs.
```

Comment:
I still don't understand why this has to be the case. You haven't pinpointed which method on Beam makes this assumption. I don't think that this assumption has to be present, in general.

Reply:
Ok, I think I finally understand what you are saying. I think this may be best said in terms of the underlying singularity functions. These are:

<x - a>**-2 : moment
<x - a>**-1 : point load 
<x - a>**0 : constant
<x - a>**1 : ramp
<x - a>**2 : quadratic
etc

The x axis is along the length of the beam and uses x as the coordinate along that axis. x=0 can be at any location relative to a physical beam's geometric span.

It is assumed that the y axis is normal to the x axis and that the forces are applied in the positive y direction (if you provide +<x-a>**n) and along the negative y direction (if you provide -<x-a>**n).

The odd thing is then what does +/-<x-a>**-2 mean in terms of direction of the moment? The moment has to be positive or negative relative to the z axis (which is normal to both x and y and creates a right handed system). We have to make an assumption about the positive sense of the moment relative to the already defined positive x and positive y directions. I think the natural thing would be to ensure that +<x-a>**-2 is always about the positive z axis. This is independent of whether you call positive y "up" or "down". (IMO, it is not helpful to use the words "up" or "down" as they are not precisely defined in this context.)

We need also to be careful to not conflate the sign conventions of the forces and moments with the definitions of the positive sense of the axes. The sign convention of a force vector that is parallel to the y axis can be that a positive force sign convention is such that it is directed in the negative y direction. One question then is, does the Beam() class assume that the sign conventions for forces and moments are tied inherently to the positive and negative senses of the coordinates axes? If it does, then we need to state that clearly.

We need to take one example and show how to solve the problem with the different combinations of the sense of the coordinate axes and the different sign conventions of the forces/moments. So that people can clearly see what is meant by "sign convention" and "positive/negative".

Those are some thoughts. Let me know if this makes sense to you.

Reply:
> We have to make an assumption about the positive sense of the moment relative to the already defined positive x and positive y directions. I think the natural thing would be to ensure that `+<x-a>**-2` is always about the positive z axis. This is independent of whether you call positive y "up" or "down".

If we have to ensure that  `+<x-a>**-2` is always about the positive z axis, we also have to ensure that the all the other loads( n>-2) acting along the negative Y direction are also written as `+<x-a>**n`. This is exactly what I have been saying. If the user is assuming moment acting along positive z axis to be of a certain sign(positive or negative), they should also ensure that all the other loads acting along negative Y direction also have the same sign as moment.

> One question then is, does the Beam() class assume that the sign conventions for forces and moments are tied inherently to the positive and negative senses of the coordinates axes? If it does, then we need to state that clearly.

The beam class does not make any such assumptions, a load acting along negative Y axis can have positive sign.